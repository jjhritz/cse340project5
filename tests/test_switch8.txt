i, j, k;
{
    i = 0;
    j = 0;
    k = 1000;

    WHILE i < 10 {
        print i;

        SWITCH i {
            CASE 0: {
                j = 100;
                print j;
            }
            CASE 3: {
                j = 200;
                print j;
            }
            CASE 5: {
                j = 300;
                print j;
            }
            DEFAULT: {
                j = 400;
                print j;
            }
        }

        i = i + 1;
    }

    print k;
}
