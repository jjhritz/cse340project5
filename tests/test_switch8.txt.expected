0
100
1
400
2
400
3
200
4
400
5
300
6
400
7
400
8
400
9
400
1000
